netdisco/discoverables/webos_tv.py

```diff
@@ -8,8 +8,12 @@ class Discoverable(SSDPDiscoverable):
     """Add support for discovering LG WebOS TV devices."""
 
     def info_from_entry(self, entry):
-        """Return the most important info from a uPnP entry."""
-        return urlparse(entry.values['location']).hostname
+        """Get most important info, which is name, model and host."""
+        name = entry.description['device']['friendlyName']
+        model = entry.description['device']['modelName']
+        host = urlparse(entry.values['location']).hostname
+
+        return name, model, host
```

Comment:
That is indeed a dict. Your syntax is wrong, each key should be surrounded by quotes.